.circleci/config.yml

```diff
@@ -0,0 +1,20 @@
+version: 2.1
+
+jobs:
+  update:
+    machine:
+      image: ubuntu-1604:202004-01
```

Comment:
We don't need machine here, right? a regular Docker image is enough?

Reply:
yes, updated that